tests/cases/conformance/expressions/objectLiterals/objectLiteralErrors.ts

```diff
@@ -15,7 +15,7 @@ var e11 = { 1.0: 0, '1': 0 };
 var e12 = { 0: 0, 0: 0 };
 var e13 = { 0: 0, 0: 0 };
 var e14 = { 0: 0, 0x0: 0 };
-var e14 = { 0: 0, 000: 0 };
+var e14 = { 0: 0, 0o0: 0 };

```

Comment:
This parser diagnostic will block the “Duplicate identifier” errors being appeared, which is the main interest of this file, thus the change.